# Decoding the stochastic profile of m6A over the entire transcriptome
## Abstract 
N6-methladenosine (m6A), an abundant eukaryotic mRNA modification, is a crucial epigenetic marker dynamically regulated by demethylase (Erasers), methyltransferase (Writers), and binding proteins (Readers). Hence, decoding the stochastic profile of m6A over transcriptome is invaluable to our understanding of ^ its biological functions^ . The m6A ^ site over 1624625 DRACH motifs on human exons were summarized from 40 ^ experiments. Four machine learning algorithms, generalized linear model (GLM), multi-layer perceptron (MLP), extreme gradient boosting (XGBoost), and random forest (RF), were implemented to build ^ Poisson regression models. Compared with^  classification models used in previous studies, our ^ model provides a new framework ^ to integrate multiple single-base RNA modification datasets. We demonstrated that the Poisson regressors can better predict the biological and technical variation between experiments than ^ classifiers trained ^ with same feature^ s. In addition, we for the first time ^ utilized the protein binding information for prediction and achieved significantly better performance (MSE X.XX / AUC X.XX) than models based on only sequence-derived and genome-derived features (MSE X.XX / AUC X.XX).

## Material and Methods
### Genomic features description
In this project, 35 genome-derived features were used in our model. Feature 1-13 are dummy variables that demonstrate whether a site is overlapped to a topological region within the transcript. Feature 14-16 describe their relative position to 5’UTR, 3’UTR and exon while feature 17-19 is the length of the corresponding 5’UTR, 3’UTR and exon region. Feature 20-22 measure the nucleotide distances to the splicing junction or nearest adenosine sites. Feature 23-26 are related to the conservation score of the sites. The rest features describe the secondary structure, genomic properties and attributes of the related genes or transcripts.
